packages/react-components/lib/lifts/lift-request-form.tsx

```diff
@@ -91,70 +110,77 @@ export const LiftRequestForm = (props: LiftRequestFormProps): JSX.Element => {
   };
 
   return (
-    <form className={classes.form} onSubmit={handleLiftRequest}>
-      <div className={classes.divForm}>
-        <Autocomplete
-          getOptionLabel={(option) => option}
-          onChange={(_, value) => setDestination(value || '')}
-          options={['', ...lift.levels]}
-          renderInput={(params) => (
-            <TextField
-              {...params}
-              label="Pick a Destination"
-              placeholder="Pick a Destination"
-              variant="outlined"
-              error={!!destinationError}
-              helperText={destinationError}
+    <Dialog open={showFormDialog} onClose={() => onClose()} fullWidth={true} maxWidth={'md'}>
```

Comment:
Might want to use the `ConfirmationDialog` (see the authz components in dashboard for an example) to keep a consistent ui/ux.